app-android/app/src/main/java/de/tutao/tutanota/Contact.kt

```diff
@@ -37,10 +55,718 @@ class Contact(private val activity: MainActivity) {
 		}
 	}
 
+	suspend fun deleteContacts(userId: String, contactId: String?) {
+		activity.getPermission(Manifest.permission.READ_CONTACTS)
```

Comment:
We use these two lines a lot. It might be worth extracting them into a private function like `getPermissions()`?